scripts/masternode.js

```diff
@@ -49,11 +49,11 @@ export default class Masternode {
        @return {Promise<Object>} The object containing masternode information for this masternode
      */
     async getFullData() {
-        const strURL = `${cNode.url}/listmasternodes?params=${this.collateralTxId}`;
+        const strURL = `/listmasternodes?params=${this.collateralTxId}`;
         try {
-            const cMasternodes = (await (await fetch(strURL)).json()).filter(
-                (m) => m.outidx === this.outidx
-            );
+            const cMasternodes = (
+                await (await retryWrapper(fetchNode, false, strURL)).json()
+            ).filter((m) => m.outidx === this.outidx);
             if (cMasternodes.length > 0) {
```

Comment:
Rather than importing retryWrapper, we should move all network functions to the Network class, and call something like getNetwork().callRPC(...) instead.
This is needed for MPW-tauri, so we can plug a TauriNetwork in getNetwork and everything should work

Reply:
Added suggestion, it's a good abstraction, much nicer RPC handling and Tauri futureproofing.